component/azstorage/config.go

```diff
@@ -385,7 +388,18 @@ func ParseAndValidateConfig(az *AzStorage, opt AzStorageOptions) error {
 	az.stConfig.cancelListForSeconds = opt.CancelListForSeconds
 
 	az.stConfig.telemetry = opt.Telemetry
-
+	az.stConfig.blobFilter = opt.BlobFilter
+	if len(opt.BlobFilter) > 0 {
+		fmt.Println(opt.BlobFilter)
+		erro := filter.ParseInp(&opt.BlobFilter)
+		fmt.Println(len(filter.GlbFilterArr))
+		fmt.Println(len(opt.BlobFilter))
+		if erro != nil {
+			log.Err("no filters applied, mount failed")
```

Comment:
For log.Debug/Err we are following a format which is consistent across all our logs. Just check other existing logs in this file and modify accordingly.